.github/workflows/artifacts-build.yml

```diff
@@ -194,7 +194,7 @@ jobs:
     - name: "⭐⭐⭐ Links to Uploaded Artifacts ⭐⭐⭐"
       run: |
         echo "::group::Wasm Demo Preview"
-        echo "https://storage.googleapis.com/${{ env.GCP_MAIN_BUCKET_ID }}/gha/wasm-demo/index.html"
+        echo "https://storage.googleapis.com/${{ env.GCP_MAIN_BUCKET_ID }}/gha/wasm-demo/dist/index.html"
```

Comment:
I think the better fix is to update the upload to

```
        gsutil -m cp -r ffi/diplomat/js/examples/wasm-demo/dist/* gs://${{ env.GCP_MAIN_BUCKET_ID }}/gha/wasm-demo
```

This way there's no nested `dist` folder inside `wasm-demo`. Otherwise the deploy to GH pages job also needs an update